TankDriver.App/Game.cs

```diff
@@ -102,9 +109,14 @@ protected override void Update(GameTime gameTime)
 			}
 
 			var mouseState = Mouse.GetState();
+			if (mouseState.LeftButton == ButtonState.Pressed) {
+				_tank.ShootInto (_bulletSpace);
+			}
```

Comment:
I think we need to add a cooldown so the tank couldn't shot too many bullets at once. Just a note to myself; let's add this later. I'll go file an issue about that.